src/App.tsx

```diff
@@ -78,27 +68,29 @@ export default function App() {
         ? 'custom-theme-override'
         : ''
 
-    function onlineToast(toastBody: JSX.Element, options) {
-        if (onlineToastRef.current && toast.isActive(onlineToastRef.current)) {
-            toast.update(onlineToastRef.current, { render: toastBody, ...options })
-        } else {
-            onlineToastRef.current = toast[options.type](toastBody, options)
+    function onlineToast(...showToastParams: Parameters<typeof ToastManager.showToast>) {
+        if (onlineToastRef.current && ToastManager.isToastActive(onlineToastRef.current)) {
+            ToastManager.dismissToast(onlineToastRef.current)
         }
```

Comment:
one else statement seems to be changed 
is this expected

Reply:
This is intentional